CHANGELOG.md

```diff
@@ -11,6 +11,95 @@ All notable changes to Botonic will be documented in this file.
+    - [Fixed TS types](https://github.com/hubtype/botonic/pull/1069)
+    - [Added `_hubtype_case_status` and `typification` fields](https://github.com/hubtype/botonic/pull/1104)
+  - [Refactored utils](https://github.com/hubtype/botonic/pull/1067) for better maintainability.
+  - [Improved webchat tests](https://github.com/hubtype/botonic/pull/1066).
+
+- [@botonic/nlu](https://www.npmjs.com/package/@botonic/nlu)
+
+  - Upgrade `tfjs` to [2.7.0](https://github.com/hubtype/botonic/commit/3d42d12562ab5196b5f8a113387a60feb90b3fed).
+
+* [@botonic/plugin-nlu](https://www.npmjs.com/package/@botonic/plugin-nlu)
+
+  - Upgrade `tfjs` to [2.7.0](https://github.com/hubtype/botonic/commit/3d42d12562ab5196b5f8a113387a60feb90b3fed).
+
+### Fixed
+
+- [Project](https://github.com/hubtype/botonic)
+
+  - Fixed [broken links](https://github.com/hubtype/botonic/pull/1037) in READMEs.
+  - Fixed [`Was this article useful?` widget](https://github.com/hubtype/botonic/commit/4d1ea02836c4f29a4d51d9aff1081afa7647dc3f).
+
+* [@botonic/cli](https://www.npmjs.com/package/@botonic/cli)
+
+  - [Fixed params request](https://github.com/hubtype/botonic/pull/1070) failing in `Oauth 3.1.0+`.
+
+- [@botonic/core](https://www.npmjs.com/package/@botonic/core)
+
+  - Now `getRoute` [checks correctly](https://github.com/hubtype/botonic/pull/903) if the input matches with a `childRoute`.
+
+* [@botonic/react](https://www.npmjs.com/package/@botonic/react)
+
+  - Added [missing storage key in constructor](https://github.com/hubtype/botonic/pull/1039), now `storageKey` works as expected.
+  - Automatically [call `onMessage`](https://github.com/hubtype/botonic/pull/1056) when receiveng messages coming from server-side.
+  - Added missing call to `stringifyWithRegexs` causing some regexes not being updated in local/session storages. Fixed react warnings (produced by `npm run test`), log deprecated props in custom messages. [#1063](https://github.com/hubtype/botonic/pull/1063)
+    Deprecated props:
+    - **`enableTimestamps` => `enabletimestamps`**
+    - **`imagesStyle` => `imagestyle`**
+  - Fixed sending of unsent inputs to be [more consistent](https://github.com/hubtype/botonic/pull/1041).
+  - [Fixed `onClose`](https://github.com/hubtype/botonic/commit/03ce2ec264181df537fd9c8c3695e362f48bfd05) being called on the very first render of the app.
+  - [Fixed](https://github.com/hubtype/botonic/pull/1108) `WebchatSettings` component not updating settings correctly.
+  - [Fixed](https://github.com/hubtype/botonic/pull/1108) `WebchatSettings` component not updating settings correctly.
+  - [Set ack to 1](https://github.com/hubtype/botonic/commit/03e2fa3cac6ee40bb5a83388a5b40b082f26d0ce) for custom user messages by default.
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+
+  - Now the csv importer of texts coming from content translator groups all the fields from the same content. [#1036](https://github.com/hubtype/botonic/pull/1036).
```

Comment:
The csv importer of texts, which was  coming from content translator, now groups all the fields from the same content.